cluster/log-dump/log-dump.sh

```diff
@@ -164,6 +165,10 @@ function save-logs() {
         for svc in "${services[@]}"; do
             log-dump-ssh "${node_name}" "sudo journalctl --output=cat -u ${svc}.service" > "${dir}/${svc}.log" || true
         done
+
+        if [[ "$dump_systemd_journal" = true ]]; then
```

Comment:
if [[ "$dump_systemd_journal" == "true" ]]; then ??

Reply:
I think my version also worked, but my bash knowledge is very limited so changed to your version :)